src/__swaps__/screens/Swap/resources/assets/userAssets.ts

```diff
@@ -86,9 +86,6 @@ export const userAssetsSetQueryData = ({ address, currency, userAssets, testnetM
 async function userAssetsQueryFunction({
   queryKey: [{ address, currency, testnetMode }],
 }: QueryFunctionArgs<typeof userAssetsQueryKey>): Promise<ParsedAssetsDictByChain> {
-  if (!address) {
```

Comment:
this function is now no longer enabled if there is no address

Reply:
could be worth keeping in case the useQuery config is overridden or userAssetsFetchQuery is used in the future (it's currently an usused export)

Reply:
yeah i'm a fan of keeping an early return here in case the query function is used in a direct fetch.